src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -178,7 +219,8 @@ private void ProcessInputFileList(ITaskItem[] inputFiles, List<ITaskItem> imageC
                 // unless an explicit PublishReadyToRunEmitSymbols flag is enabled by the app developer. There is also another way to profile that the runtime supports, which does
                 // not rely on the native PDBs/Map files, so creating them is really an opt-in option, typically used by advanced users.
                 // For debugging, only the IL PDBs are required.
-                if (EmitSymbols)
+                // TODO: Need to figure out if we'll add the native symbol emission feature to crossgen2, or use crossgen to generate them
```

Comment:
There's a conversation going on around the native symbols feature. Most likely, it's looking like we'll use crossgen for it, as this won't be implemented in crossgen2 for the .NET 5.0 timeframe